pyproject.toml

```diff
@@ -124,6 +124,7 @@ module = [
   # https://github.com/briancurtin/deprecation/issues/56 - Add type information (PEP 561)
   "deprecation.*",
   "grpc.framework.foundation.*",
+  "hightime.*",
```

Comment:
Please add a comment linking to https://github.com/ni/hightime/issues/4

Reply:
Sure, added the corresponding link.